<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientSelect</class>
 <widget class="QWidget" name="PatientSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
border:none;
background:#FFFFFF;
font-size:16px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
border:none;
background:#FFFFF0;
}


QWidget:disabled{
background:#C4C4C4;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight:700;
color:#333333;
font-size:18px;
line-height:16px;</string>
           </property>
           <property name="text">
            <string>起始日期：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="lineedit_date_start">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>110</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
background:#F0F8FF;
}


QLineEdit:disabled{
background:#D3D3D3;
}</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_date_start">
              <property name="minimumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background-image: url(:/icons/images/calendar.png);
background-position:center;
border:none;
background-repeat:no-repeat;
}

QPushButton:hover{
background: #40a9ff;
background-image: url(:/icons/images/calendar.png);
background-position:center;
border:none;
background-repeat:no-repeat;
}

QPushButton:pressed{
background:#4EEE94;
background-image: url(:/icons/images/calendar.png);
background-position:center;
border:none;
background-repeat:no-repeat;

}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight:700;
color:#333333;
font-size:18px;
line-height:16px;</string>
           </property>
           <property name="text">
            <string>截止日期：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="lineedit_date_end">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>110</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
background:#F0F8FF;
}


QLineEdit:disabled{
background:#D3D3D3;
}</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_date_end">
              <property name="minimumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background-image: url(:/icons/images/calendar.png);
background-position:center;
border:none;
background-repeat:no-repeat;
}

QPushButton:hover{
background: #40a9ff;
background-image: url(:/icons/images/calendar.png);
background-position:center;
border:none;
background-repeat:no-repeat;
}

QPushButton:pressed{
background:#4EEE94;
background-image: url(:/icons/images/calendar.png);
background-position:center;
border:none;
background-repeat:no-repeat;
}

</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>373</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
background:#FFFFF0;
border:none;
}


QWidget:disabled{
background:#C4C4C4;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="0,0,0,1">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QLabel{
color:#1a1a1a;
font-size:16px;
line-height:24px;
}

QLineEdit{
color:#1a1a1a;
font-size:16px;
line-height:24px;
padding-left:8px;
border:none;
border-bottom:1px solid #1a1a1a;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font-weight:700;
color:#333333;
font-size:18px;
line-height:16px;</string>
           </property>
           <property name="text">
            <string>病人ID：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">font-weight:700;
color:#333333;
font-size:18px;
line-height:16px;</string>
              </property>
              <property name="text">
               <string>性别：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="styleSheet">
               <string notr="true">font-weight:700;
color:#333333;
font-size:18px;
line-height:16px;</string>
              </property>
              <property name="text">
               <string>男</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="styleSheet">
               <string notr="true">font-weight:700;
color:#333333;
font-size:18px;
line-height:16px;</string>
              </property>
              <property name="text">
               <string>女</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_query">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background:#40a9ff;
border:1px solid;
border-color:#666666;
color:#ffffff;
font-size:18px;
line-height:24px;
}

QPushButton:pressed{
background:#4EEE94;
}

QPushButton:disabled{
background:#D3D3D3;
}

</string>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_query_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background:#40a9ff;
border:1px solid;
border-color:#666666;
color:#ffffff;
font-size:18px;
line-height:24px;
}

QPushButton:pressed{
background:#4EEE94;
}

QPushButton:disabled{
background:#D3D3D3;
}
</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">border:1px solid #FAFAD2;
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,0">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="CustomTableView" name="tableView">
        <property name="styleSheet">
         <string notr="true">font-weight:700;
color:#333333;
font-size:18px;
line-height:16px;
background:#F0F8FF;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight:700;
color:#333333;
font-size:18px;
line-height:16px;
background:#FFF5EE;</string>
           </property>
           <property name="text">
            <string>所选择的病人：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="styleSheet">
            <string notr="true">background:#FFDAB9;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTableView</class>
   <extends>QTableView</extends>
   <header>widgets/customtableview/customtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../app.qrc"/>
 </resources>
 <connections/>
</ui>
